dglgo/dglgo/pipeline/graphpred/graphpred.jinja-py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import sklearn
+import torch
+import torch.nn as nn
+from dgl.data import AsGraphPredDataset
+from dgl.dataloading import GraphDataLoader
+from torch.optim.lr_scheduler import ReduceLROnPlateau
+from tqdm import tqdm
+{{ data_import_code }}
+
+{{ model_code }}
+
+def train(device, loader, model, criterion, optimizer):
+    model.train()
+
+    for _, (g, labels) in enumerate(tqdm(loader, desc="Iteration")):
+        g = g.to(device)
+        labels = labels.reshape(g.batch_size, -1).to(device)
```

Comment:
Why need a `reshape` here?

Reply:
Previously this was for handling labels of a CSV dataset. This is now handled by `CSVDataset`.